tests/lib/test_openstack_hypervisor.py

```diff
@@ -101,14 +102,38 @@ def test_search_all_hvs(self):
         """
         Tests calling search_all_hvs
         """
-        self.instance.search_all_hvs(cloud_account="test")
 
-        self.mocked_connection.assert_called_once_with("test")
+        result = self.instance.search_all_hvs(self.cloud_account)
 
-        self.api.list_hypervisors.assert_called_once_with(
-            filters={},
+        self.mocked_connection.assert_called_once_with(cloud_name=self.cloud_account)
+        self.assertEqual(result, self.api.list_hypervisors.return_value)
+
+    def test_get_all_empty_hvs_without_disabled_hvs(self):
```

Comment:
We need tests that `vcpus_user > 0` and `running_vms > 0` correctly filter out those HVs instead of returning them as eligible